src/main/common/utils/isNonCMCReference.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks if given reference number is not a CMC reference number.
+ *
+ * @param {string} referenceNumber the reference number
+ * @returns {boolean} true if the number if not a CMC reference number, false otherwise
+ */
+export function isNonCMCReference (referenceNumber?: string): boolean {
```

Comment:
not hugely sure if this function is really needed, there's already a validator for the format, the only reason not to just use the validator would be it handles it being empty,

Consider refactoring this but 👍

Reply:
Refactored by removing `@Matches` from `ClaimReference` model.